src/components/Search.js

```diff
@@ -0,0 +1,25 @@
+import useStore from '../hooks/useStore';
+
+import {SearchBar, SearchContainer, SearchInput} from './styled/Search.styled';
+
+export default function Search() {
+	const setSearchItem = useStore(state => state.setSearchItem);
+	const searchItem = useStore(state => state.searchItem);
+
+	return (
+		<SearchBar>
+			<SearchContainer>
+				<SearchInput
+					value={searchItem}
+					onChange={event => {
+						event.preventDefault();
+						const search = event.target.value;
+						setSearchItem(search);
+					}}
+					type="search"
+					name="search Ingredient"
+				></SearchInput>
```

Comment:
Your `input` element does not need a closing tag. But it needs a `label` or a at least an `aria-label`. Please change this for a better accessibility of your search form.